askama_derive/src/parser/expr.rs

```diff
@@ -182,8 +182,8 @@ fn expr_single(i: &str) -> IResult<&str, Expr<'_>> {
         expr_num_lit,
         expr_str_lit,
         expr_char_lit,
-        expr_path,
         expr_rust_macro,
+        expr_path,
```

Comment:
I actually dislike this reordering, because it means that the stream is parsed three times for the average case, i.e. one is not invoking a macro, and one is utilizing an identifier/variable instead of a path. I think it would be best to implement parsing macros inside of `expr_suffix` anyway. We just need to make sure that the left-hand side is a path or a macro.

Reply:
It might get messy. Something has to accept `path::macro!` and `macro!` or `!(..)` as an expression to parse down the line. I did try adding the capability to identifiers and paths to accept a macro, but it was very fragile. Maybe the better idea is to fix the ordering instead?

Reply:
It does this now 👍